packages/react-resizable-panels-theme/stories/react-resizable-panels-theme.stories.tsx

```diff
@@ -0,0 +1,399 @@
+    </div>
+  );
+}
+
+export function Border() {
+  return (
+    <div className="react-resizable-panels-theme-salt">
+      <FlexLayout className="box">
+        <PanelGroup direction="horizontal">
+          <Panel id="left" minSize={0} defaultSize={25} className="center">
+            <Text>Left</Text>
+          </Panel>
+          <PanelResizeHandle className="resize-handle-border-right" />
+          <Panel minSize={50} className="center">
+            <Text>Center</Text>
+          </Panel>
+          <PanelResizeHandle className="resize-handle-border-left" />
+          <Panel minSize={0} defaultSize={25} className="center">
+            <Text>Right</Text>
+          </Panel>
+        </PanelGroup>
+      </FlexLayout>
+    </div>
+  );
+}
+
+export function Variant() {
+  return (
+    <div className="react-resizable-panels-theme-salt">
+      <FlexLayout className="box">
+        <PanelGroup direction="horizontal">
+          <Panel
+            minSize={0}
+            defaultSize={25}
+            className="center resizable-panel-salt-variant-secondary"
+          >
+            <Text>Left</Text>
+          </Panel>
+          <PanelResizeHandle className="resize-handle-border-right resize-handle-salt-variant-secondary" />
+          <Panel minSize={50} className="center">
+            <Text>Center</Text>
+          </Panel>
+          <PanelResizeHandle className="resize-handle-border-left resize-handle-salt-variant-tertiary" />
+          <Panel
+            minSize={0}
+            defaultSize={25}
+            className="center resizable-panel-salt-variant-tertiary"
+          >
+            <Text>Right</Text>
+          </Panel>
+        </PanelGroup>
+      </FlexLayout>
+    </div>
+  );
+}
+
+export function Size() {
+  return (
+    <div className="react-resizable-panels-theme-salt">
+      <FlexLayout className="box">
+        <PanelGroup direction="horizontal">
+          <Panel minSize={20} className="center">
+            <Text>Left [20%, X]</Text>
+          </Panel>
+          <PanelResizeHandle className="resize-handle-border-left resize-handle-border-right" />
+          <Panel id="middle" minSize={40} maxSize={60} className="center">
+            <Text>Middle [30%, 60%]</Text>
+          </Panel>
+          <PanelResizeHandle className="resize-handle-border-left resize-handle-border-right" />
+          <Panel minSize={20} className="center">
+            <Text>Right [20%, X]</Text>
+          </Panel>
+        </PanelGroup>
+      </FlexLayout>
+    </div>
+  );
+}
+
+export function CollapsibleSetSize() {
+  return (
+    <div className="react-resizable-panels-theme-salt">
+      <FlexLayout className="box">
+        <PanelGroup direction="horizontal">
+          <Panel
+            collapsible
+            collapsedSize={15}
+            minSize={30}
+            maxSize={30}
+            className="center"
+          >
+            <Text>
+              Left <br />
+              {"{10%, 30%}"}
```

Comment:
Nit: The above story with percentages have square brackets and this one has curly ones.
```suggestion
              [10%, 30%]
```

Reply:
Will make the change as I need to run chromatic again